docs/content/docs/3.options/2.routing.md

```diff
@@ -131,7 +131,7 @@ Set to a path to which you want to redirect users accessing the root URL (`/`).
 - default: `false`
 
 ::callout{type=warning}
-The `dynamicRouteParams` is deprecated and will be removed in `v8.1`, use the [`useSetI18nParams`](/docs/api#useseti18nparams) composable instead.
+The `dynamicRouteParams` is deprecated and was removed in `v8.1`, use the [`useSetI18nParams`](/docs/api#useseti18nparams) composable instead.
```

Comment:
Was it ? I don't see anything on the changelog for 8.1

Reply:
You're right, we decided to delay the removal to v9 instead since it's a breaking change, we should change the warnings and docs to reflect that 😅.

See https://github.com/nuxt-modules/i18n/pull/2801 and roadmap discussion https://github.com/nuxt-modules/i18n/discussions/2814.